src/ui/components/Toolbar.tsx

```diff
@@ -147,6 +147,13 @@ export default function Toolbar() {
           </>
         ) : null}
         {logProtocol ? <ToolbarButton icon="code" label="Protocol" name="protocol" /> : null}
+
+        {sidePanelToggle !== null && (
+          <>
+            <div className="grow"></div>
+            <div className="relative px-2">{sidePanelToggle}</div>
+          </>
+        )}
```

Comment:
Not sure if we need the extra button or if the vs code ux where you click the active button to toggle it is fine?

My personal preference is that we should bring back the vs code ux and remove the sidebar toggle button. But, I'd be open to re-adding the button if we see users struggle to collapse the sidebar.

Reply:
Jon and I agreed this button was a nice addition.